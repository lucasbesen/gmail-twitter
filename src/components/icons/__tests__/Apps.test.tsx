import React from 'react';
import renderer from 'react-test-renderer';

import { Apps } from '../';

it('should render Apps icon without error', () => {
  const component = renderer.create(<Apps />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
