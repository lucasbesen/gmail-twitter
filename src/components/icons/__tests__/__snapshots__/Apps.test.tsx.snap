// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Apps icon without error 1`] = `
<svg
  aria-hidden="true"
  className="sc-bdVaJa euMJkF StyledIconBase-sc-bdy9j4 jbPeIR"
  fill="currentColor"
  focusable="false"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M4 8h4V4H4v4zm6 12h4v-4h-4v4zm-6 0h4v-4H4v4zm0-6h4v-4H4v4zm6 0h4v-4h-4v4zm6-10v4h4V4h-4zm-6 4h4V4h-4v4zm6 6h4v-4h-4v4zm0 6h4v-4h-4v4z"
  />
</svg>
`;
